reviews/2022-cdk8s-annual.md

```diff
@@ -0,0 +1,101 @@
+## Background
+
+CDK For Kubernetes (cdk8s) is an open-source software development framework for defining Kubernetes 
+applications and reusable abstractions. Users leverage familiar programming languages and rich object-oriented APIs to 
+create standard Kubernetes manifests which can be applied to any Kubernetes cluster.
+
+CDK8s joined the CNCF on November 10, 2022.
+
+## Annual Review
+
+The following contains the requested content for the annual review, as described [here](https://github.com/cncf/toc/blob/main/process/sandbox-annual-review.md).
+
+### Include a link to your project’s devstats page.
+
+The full devstats page for cdk8s is available here: https://cdk8s.devstats.cncf.io/d/8/dashboards
+
+* Maintaining a steady rate of merged PRs across our repositories. 
```

Comment:
what is the rate of merged PRs? we should extract meaningful stats from the dashbaord

Reply:
Yeah it has been hard to do because the numbers there don't really add up with respect to automated commits. I'll just drop this.